pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +695,20 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+
+			if len(cr.Spec.Objects.Servers) > 0 {
+				server := cr.Spec.Objects.Servers[0]
+				// if the configuration use only one replica we increment
+				if server.JBPMCluster && *serverReplicas == 1 {
```

Comment:
why not just use the existing `serverSet` instead of creating a new server object?